keylime/revocation_notifier.py

```diff
@@ -79,6 +80,36 @@ def worker(tosend):
     t.start()
 
 
+def notify_webhook(tosend):
+    url = config.get('cloud_verifier', 'webhook_url')
```

Comment:
Also here, in case one has an older keylime.conf, this probably throws an error. We need the third parameter providing the default ('').

Reply:
@THS-on please address @stefanberger comment, so I can test and merge

Reply:
I'll do that. Do we want to merge the feature as is and replace it in the future with something more generic 
or should we keep this just open as a reference?

Reply:
Let's merge this one, and re-work later.

Reply:
Ok, all requested changes should now be implemented.